src/EditorFeatures/CSharpTest/UsePatternMatching/CSharpAsAndNullCheckTests.cs

```diff
@@ -461,5 +461,49 @@ void M(object y)
     }
 }", compareTokens: false);
         }
+
+        [WorkItem(17129, "https://github.com/dotnet/roslyn/issues/17129")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTypeCheck)]
+        public async Task TestTrivia2()
```

Comment:
It's really that the indentation looks appropriate at the end.  Essentially this test is explicitly stating that it cares about *all* trivia, and that if anything changes in that regard, it needs to be looked at. i.e. as opposed to 99.999% of the rest of the CodeAction tests that do not care to test what sort of end formatting they get.